test/test_model_anon.py

```diff
@@ -15,19 +15,22 @@
 from src.Model.Anon import (
     _check_identity_mapping_file_exists,
     _create_reidentification_spreadsheet,
-    _has_child_CSV_directory,
+    _trim_bracketing_single_quotes,
```

Comment:
eliminated testing of the _has_child_CSV_directory() function because the function was eliminated in favour of the one-line python os.path.exists() method.